homeassistant/components/tessie/sensor.py

```diff
@@ -210,4 +210,7 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the state of the sensor."""
+        if isinstance(self.value, str):
+            # Make value lower case for translation
+            self.value.lower()
```

Comment:
It probably makes sense to have a `value_fn` instead

Reply:
`trafikverket_ferry` has an example

Reply:
I thought about this, but all string values from the API will be translated. But if you think a lambda is better ill do that.

Reply:
The lambda makes it a lot more clear whats going on. With lower casing all the `str` is not clear why without a comment.  Either is fine (comment or lambda)

Reply:
Ill add the lambda